### PR TITLE
[MM-50460]  - Fixed textarea focus

### PR DESCRIPTION
**Pull request by masif2002**
#### Summary

This PR fixes the bug of not being able to focus on the textbox when clicked anywhere above the “Aa”, attachment, emoji, or send buttons. Right now, if you click anywhere in the textbox, the focus goes to it. (As expected)

* #### Before
    before
* #### After
    after

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/22696
JIRA: https://mattermost.atlassian.net/browse/MM-50460 

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by AshishDhama — CHANGES_REQUESTED** [review]
This fix will break the edge case it was there for in first place, could you please look for some other solutions

**Comment by masif2002** [comment]
> This fix will break the edge it was there for in first place, could you please look for some other solutions

Sure, i'll take a look

File: webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx [hunk 1]
```diff
@@ -378,10 +379,8 @@ const AdvanceTextEditor = ({
         const currentWidth = width + inputPaddingX;
 
         if (currentWidth >= maxWidth) {
-            input.style.maxWidth = '100%';
```
[the remark this answers is not in the document]
Comment (masif2002) [reply webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx:381]: > @masif2002 I am seeing this after your changes, Screenshot 2023-04-03 at 3 23 23 PM The edge case for which we are setting maxWidth like this.

Can you please let me know how i can reproduce what is shown in this screenshot?

[the remark this answers is not in the document]
Comment (LeonardJouve) [reply webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx:381]: @masif2002 To reproduce, you must hide the formatting bar ('Aa' button) and write a long message in the channel input

**Comment by francislavoie** [comment]
@masif2002 any chance you can follow up on this? I'm bothered by this every day :sweat_smile: so I'd love to see a fix for this

**Comment by M-ZubairAhmed** [comment]
We are taking this internally to fix

**Pull request closed without merging**